yarn.lock

```diff
@@ -4,44 +4,32 @@
 
 "@adobe/css-tools@^4.0.1":
   version "4.2.0"
-  resolved "https://registry.yarnpkg.com/@adobe/css-tools/-/css-tools-4.2.0.tgz#e1a84fca468f4b337816fcb7f0964beb620ba855"
+  resolved "https://registry.npmjs.org/@adobe/css-tools/-/css-tools-4.2.0.tgz"
```

Comment:
It looks to me like you are using `npm` instead of `yarn`, which is what is causing inconsistent updates - and unwanted difffs - when you update your dependencies.

I believe that `npm` warns to say that only one package manager at a time should be used...

Could this change to `yarn.lock` be reverted please...

And I suggest next time running `yarn install` or even vanilla `yarn` to update dependencies, and you will then get exactly the correct versions, which should give you a smoother dev experience...

Reply:
Sorry about that, in my still-short history with Javascript I was actually never quite sure when I should use Yarn and when NPM. I should clarify that with myself...
Reverted.

Reply:
No problem at all. Thanks for fixing it.